activiti-cloud-audit-service/activiti-cloud-services-audit-jpa/src/main/resources/META-INF/application.properties

```diff
@@ -1,2 +1,4 @@
 # enable clean up endpoints
-activiti.enable-clean-up=true
\ No newline at end of file
+activiti.enable-clean-up=true
+
+activiti.cloud.audit.uri-prefix=
```

Comment:
I think the default empty value of the uri prefix should be provided in dedicated properties source for each service, i.e. `activiti-cloud-audit-service.properties`. The application.properties will be overriden by example Spring Boot applicattion properties.

Reply:
After your suggestion, I did a review of properties files, now we have 4 properties in Spring Boot applicattion properties:

- activiti.cloud.audit.uri-prefix=/audit
- activiti.cloud.query.uri-prefix=/query
- activiti.cloud.notification.uri-prefix=/notifications
- activiti.cloud.notification.graphiql.uri-prefix=/notifications

This means that we have 4 properties files with default values:

- activiti-cloud-audit-service.properties
- activiti-cloud-query-service.properties
- activiti-cloud-notification-service.properties
- activiti-cloud-notification-graphiql.properties

Reply:
we removed these properties